Samples/Samples/ViewModel/CalendarEventAddViewModel.cs

```diff
@@ -8,6 +8,7 @@
 using System.Windows.Input;
 using Xamarin.Essentials;
 using Xamarin.Forms;
+using PropertyChangingEventArgs = System.ComponentModel.PropertyChangingEventArgs;
```

Comment:
why is this necessary? can't we just have
using System.ComponentModel

Reply:
Not required anymore will remove